### PR TITLE
VTCCA-5095 - Added in formatted rateable value - flaky test possibility

### PR DESCRIPTION
**Pull request by WitchellJack**

### CODE REVIEW

File: app/views/dvr/dvrFiles.scala.html [hunk 1]
```diff
@@ -34,10 +34,11 @@
   govukSummaryList: GovukSummaryList,
   govukRadios: GovukRadios,
   govukErrorSummary: GovukErrorSummary,
-  formWithCSRF: FormWithCSRF
+  formWithCSRF: FormWithCSRF,
+  govukInsetText: GovukInsetText
 )
 
-@(model: AvailableRequestDetailedValuation, startCheckForm: Form[StartCheckForm])(implicit request: Request[_], messages: Messages, config: ApplicationConfig)
+@(model: AvailableRequestDetailedValuation, startCheckForm: Form[StartCheckForm], rateableValue: Option[Long])(implicit request: Request[_], messages: Messages, config: ApplicationConfig)
```
Comment (habeeb01) [inline app/views/dvr/dvrFiles.scala.html:41]: can you add `rateableValue` to the `model` instead please.

Comment (PeterPerhac) [reply app/views/dvr/dvrFiles.scala.html]: yes, please, well spotted Habeeb

Comment (WitchellJack) [reply app/views/dvr/dvrFiles.scala.html]: Yeah I just had a call with habeeb as my commit didn't answer this at all - it seems my brain really still hasn't switched on 😄

**Comment by habeeb01** [comment]
LGTM 👍

**Comment by PeterPerhac** [comment]
👍  looks good, just need to run it through scalafmt first

**Pull request merged**